static/include/components/motor-sensor.md

```diff
@@ -0,0 +1,13 @@
+If you want to control your motor by specifying the distance and velocity in terms of distance and distance/time, for example 2 rotations at 5 m/s and you have an encoder, you can make use of the position of your encoder to adjust the distance and velocity.
+
+One way to adjust distance and velocity based on an encoder's measurements, is using a PID control loop that measures the current position and change in position and determines the difference or error from the desired distance and velocity to compute a correction for the motor based on three terms:
```

Comment:
```suggestion
The 'gpio' model of Motor for Encoded Motors uses a PID control loop that measures the current position and change in position and determines the difference or error from the desired distance and velocity to compute a correction for the motor based on three terms:
```

Reply:
@martha-johnston , confirming that for encoded gpio motors, PID control is now always used (and the "10% increment" form of control is never used)?

Reply:
no, if you don't add the control_parameters the old version is used (the 10% thing)

Reply:
Ah right, sorry I got confused between gpio motor and Spin 🤦‍♀️ 

Makes sense, updating the suggested edit above accordingly

Reply:
updated